llama-index-integrations/readers/llama-index-readers-web/llama_index/readers/web/firecrawl_web/base.py

```diff
@@ -24,6 +25,7 @@ class FireCrawlWebReader(BasePydanticReader):
 
     firecrawl: Optional[object] = Field(None)
     api_key: str
+    api_url: Optional[str]
```

Comment:
since api_url is never passed into `super(...)`, this class attribute is never set